frontend/scss/components/organisms/resources-grid.scss

```diff
@@ -16,66 +16,59 @@
 @import 'components/atoms/_text.scss';
 
 .#{utility('resources-grid')} {
-  display: flex;
-  flex-direction: column;
-  flex-wrap: wrap;
-  justify-content: space-between;
-  margin: 40px 0;
-
-  @media (min-width: 600px) {
-    flex-direction: row;
-  }
-  @media (min-width: 768px) {
-    flex-direction: column;
-  }
-  @media (min-width: 930px) {
-    margin: 0 60px 30px 60px;
-  }
-  @media (min-width: 1024px) {
-    flex-direction: row;
-  }
+  grid-column: 3 / 23;
 }
 .#{organism('resources-grid')} {
-  &-text {
-    width: 100%;
-    padding-bottom: 20px;
+  &-cards {
+    display: grid;
+    grid-gap: 0;
+    grid-column-gap: 30px;
+    grid-template-columns: repeat(12, 1fr);
   }
 
   &-card {
-    margin: 10px 0;
-    padding: 10px 20px;
+    margin: 15px 0;
+    padding: 20px 20px;
     border-radius: 10px;
     background-color: color('white');
-    box-shadow: 6px 9px 40px 0px rgba(0,0,0,0.07);
+    box-shadow: 6px 9px 40px 0 rgba(0, 0, 0, 0.15);
```

Comment:
Please use `transparentize(color('white'), 0.85)` instead of specific color values like `rgba(0, 0, 0, 0.15);`

Reply:
changed